util/OptionsDB.h

```diff
@@ -309,6 +310,10 @@ class FO_COMMON_API OptionsDB
       * the command line */
     void        SetFromCommandLine(const std::vector<std::string>& args);
 
+    /** if the xml file exists and is not an outdated version, load the values
```

Comment:
No description of what this function does, nor the parameters nor the point of the default value of version.